src/test/java/bench/BenchmarkRunner.java

```diff
@@ -0,0 +1,19 @@
+package bench;
+
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+public class BenchmarkRunner {
+
+    public static void main(String[] args) throws RunnerException {
+
+        Options options = new OptionsBuilder()
+                .include(BenchmarkRandom10.class.getSimpleName())
```

Comment:
If you add another class containing benchmarks, make sure to add it here following the same pattern.